### DISCUSSION TITLE
ExpressLRS E32-900T30D

### DISCUSSION THREAD

**Discussion by singla2602** [ExpressLRS/ExpressLRS#489] [General]
E32-900T30D is a LoRa 1276/1278 by CDEBYTE with MCU & output power of 1W(30dbm).
Please add this module and make an update. This module is cheap, there's no need for making pcb and soldering the small components. Only firmware needs to be flashed directly. 
Also, this module can be used as TX & RX both. Therefore, we will have TX & RX with 1W(30dbm) power output.
Search "QCZEK" on youtube. QCZEK named channel guy made a LRS system which works as control link and sends & receives telemetry.
Please add E32-900T30D hardware in EXPRESSLRS. I'll be thankful to you.

**Comment by nayana1992** [comment]
it is better to post this on discord
